src/controls/polyControl.ts

```diff
@@ -50,20 +46,15 @@ export const polyActionHandler = (
   x: number,
   y: number
 ) => {
-  const poly = transform.target as Polyline,
-    pointIndex = transform.pointIndex,
-    mouseLocalPosition = getLocalPoint(transform, CENTER, CENTER, x, y),
-    polygonBaseSize = getSize(poly),
-    size = poly._getTransformedDimensions(),
-    sizeFactor = polygonBaseSize.divide(size),
-    adjustFlip = new Point(poly.flipX ? -1 : 1, poly.flipY ? -1 : 1);
-
-  const finalPointPosition = mouseLocalPosition
-    .multiply(adjustFlip)
-    .multiply(sizeFactor)
-    .add(poly.pathOffset);
+  const { target, pointIndex } = transform;
+  const poly = target as Polyline;
+  const mouseLocalPosition = sendPointToPlane(
+    new Point(x, y),
+    undefined,
+    poly.calcTransformMatrix()
```

Comment:
@ShaMan123  i do remember that x,y are always in the scene coordinates, but i m not sure that this is true also for when we run an action inside an interactive group. Do you remember if that was changed to accomodate interactive groups?

If those are in parent coordinates i have to change this with calcOwnTransform()

Reply:
I am very surprised to see x,y in the parent plane and not in the scene plane
bad decision I made at that point to limit the amount of changes

<img width="1512" alt="Screenshot 2024-01-06 at 20 04 17" src="https://github.com/fabricjs/fabric.js/assets/34343793/003e372c-601e-4b24-a32a-7e34d722bd59">

Reply:
yeah we talked about this, but i wanted to be sure. So i ll use calcOwnMatrix